### ISSUE TITLE
Something in this version seems to break multiple HandDraws on the same figure

### ISSUE THREAD

**Issue by mpacer** [bloomberg/bqplot#517]
I have a Figure where I have a radio button that determines which of two HandDraw objects are supposed to be set as the interactive layer for the Figure.

In ipywidgets 6.0 and bqplot 0.9 this works well!

![good_multiline_switching_2](https://user-images.githubusercontent.com/2482408/28644870-1bf1b31a-7210-11e7-878c-b9618423108f.gif)

But on the alpha of bqplot and the beta of ipython 7.0 this usage model no longer works:

![bad_multiline_switching_2](https://user-images.githubusercontent.com/2482408/28644819-ea9c3a88-720f-11e7-833e-1d9463825021.gif)

**Comment by mpacer** [comment]
Here are the key parts of the code making that work (happy to provide more if someone wants me to) :

```python
    draw_mode = widgets.RadioButtons(
        options=['login', 'logout'],
        value='login',
        description='Curve drawing',
        disabled=False
    )
    xax = Axis(scale=xs_hd, label='Second', grid_lines='none',
               tick_format='%M %S')
    yax = Axis(scale=ys_hd, label='Numer of Users',
               orientation='vertical', grid_lines='none')
    fig = Figure(marks=[line_login, line_logout],
                 axes=[xax, yax], interaction=handdraw_in)

    def _update_line(change):
        if draw_mode.value=='login':
            fig.interaction = None
            fig.interaction = handdraw_in
        elif draw_mode.value=='logout':
            fig.interaction = None
            fig.interaction = handdraw_out

    draw_mode.observe(_update_line)

```

**Comment by jasongrout** [comment]
What exact versions of ipywidgets and bqplot do you have? `ipywidgets.__version__` and `bqplot.__version__`.

**Comment by mpacer** [comment]
Sorry I miswrote that… edited to correct 

works: ipywidgets 6.0.0, bqplot 0.9.0
does not work: ipywidgets 7.0.0b2, bqplot 0.10.0a3

I'm currently building off of master

**Comment by ssunkara1** [comment]
Sorry didn't know that referencing this issue in a PR with the word "fixes" automatically closes the issue. Reopening it to make sure it works for @mpacer

**Comment by dmadeka** [comment]
@mpacer mentioned it seems fixed!

**Comment by mpacer** [comment]
Yes sorry thought that saying that it fixed #517 in the comment on #518 would close this. 

My latest guess on how to predict this kind of behaviour is that kind of thing only works for people with commit rights.
